eras/byron/ledger/executable-spec/src/Byron/Spec/Ledger/Update.hs

```diff
@@ -232,9 +233,13 @@ data UProp = UProp
   }
   deriving (Eq, Generic, Show, Hashable, Data, Typeable, NoThunks)
 
+#if MIN_VERSION_hashable(1,3,4)
+-- Instance Hashable (Set a) is provided by the hashable package
+#else
```

Comment:
```suggestion
-- Instance Hashable (Set a) is provided by the hashable package
#if !MIN_VERSION_hashable(1,3,4)
```

Reply:
If put the comment inside the `if` because it has an implied:

"Instance Hashable (Set a) is provided by the hashable package from 1.3.4.0 onwards so we don't need to define the instance in this case"

Perhaps I should keep it in the `if` but make it more explicit?

Reply:
Don't worry about it. It is not terribly important. If you really prefer the `if` comment `else` then let's keep it.